tests/phpunit/tests/image/editorImagick.php

```diff
@@ -816,4 +818,58 @@ public static function data_resizes_are_small_for_16bit_images() {
 			),
 		);
 	}
+
+	/**
+	 * Tests that the 'png:IHDR.color-type-orig' property is preserved after resizing.
```

Comment:
maybe add some additional details to the comment about what this header is used for, eg. "Used to identify indexed PNG images, see https://www.w3.org/TR/PNG-Chunks.html#C.IHDR"